.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
+    "configurations": [
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug MockServer",
+            "program": "${workspaceFolder:swift-aws-lambda-runtime}/.build/debug/MockServer",
+            "args": [],
+            "cwd": "${workspaceFolder:swift-aws-lambda-runtime}",
+            "preLaunchTask": "swift: Build Debug MockServer"
+        },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Release MockServer",
+            "program": "${workspaceFolder:swift-aws-lambda-runtime}/.build/release/MockServer",
+            "args": [],
+            "cwd": "${workspaceFolder:swift-aws-lambda-runtime}",
+            "preLaunchTask": "swift: Build Release MockServer"
+        },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Test swift-aws-lambda-runtime",
+            "program": "/Applications/Xcode.app/Contents/Developer/usr/bin/xctest",
+            "args": [
+                ".build/debug/swift-aws-lambda-runtimePackageTests.xctest"
+            ],
+            "cwd": "${workspaceFolder:swift-aws-lambda-runtime}",
+            "preLaunchTask": "swift: Build All"
+        }
+    ]
+}
```

Comment:
.gitignore ?

Reply:
I removed it from the commit for now. Do you want me to add it to .gitignore?